### PR TITLE
feat(browser-starfish): add ability to query for http.response_content_length in discover

### PR DESCRIPTION
**Pull request by DominikB2014**
Now that we are ingesting http.response_content_length from resource spans see (https://github.com/getsentry/relay/pull/2578/files), we need to be able to query for its avg timeseries in discover.

fyi, i'm not entirely sure if this is all the changes needed to do so, please lmk if there's anything missed.

### CODE REVIEW

**Review by AbhiPrasad — COMMENTED** [review]
Can we add a test?

**Comment by DominikB2014** [comment]
> Can we add a test?

@AbhiPrasad good idea!

File: tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py [hunk 1]
```diff
@@ -192,6 +192,38 @@ def test_top_events(self):
         assert "bar" in response.data
         assert response.data["Other"]["meta"]["dataset"] == "spansMetrics"
 
+    def test_resource_size(self):
+        for i in range(10):
```
Comment (wmak) [inline tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py:196]: nit: do we need 10 for this test case? We don't need to test the average function in snuba, so even 1 event here is enough

**Review by wmak — APPROVED** [review]
1 nit, otherwise lgtm

File: tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py (hunk 1 above)
Comment (DominikB2014) [reply tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py:196]: Yeah that's a good point, if avg function is already tested outside of this we're good. I'll refactor so its only 1 event.

**Pull request merged**